### PR TITLE
Nuri/Fix : redirect user to mt5#demo link after click the button

### PR DESCRIPTION
**Pull request by njazuli-deriv**
Changes:

-  Added dmt5#demo links
-  Redirect user to mt5#demo link after click 'Create a demo DMT5 account' button in Margin Trading page.

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [X] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by kevinw-binary — DISMISSED** [review]

File: src/pages/trade-types/margin/_start-trading.js [hunk 1]
```diff
@@ -32,7 +33,7 @@ const StartTrading = () => {
                             )}
                         </Timeline.Item>
                     </Timeline>
-                    <LinkButton mt="4rem" to="/dmt5/" secondary="true">
+                    <LinkButton mt="4rem" to={deriv_mt5_demo_app_url} secondary="true">
```
Comment (sean-binary) [inline src/pages/trade-types/margin/_start-trading.js:36]: This won't work with the new configuration of LocalizedLink as it's an external link.

`<LinkButton mt="4rem" external="true" type="deriv_app" to="/mt5#demo" target="_blank" rel="noopener noreferrer" secondary="true">` should be the right use case. Is that right @yashim-deriv?

Comment (yashim-deriv) [reply src/pages/trade-types/margin/_start-trading.js:36]: He can use directly the `type="mt5"`

```
<LinkButton mt="4rem" external="true" type="mt5" to="#demo" target="_blank" rel="noopener noreferrer" secondary="true">
```

Comment (njazuli-deriv) [reply src/pages/trade-types/margin/_start-trading.js]: followed latest LocalizedLink setup in the current commit.

File: src/pages/trade-types/margin/_start-trading.js [hunk 2]
```diff
@@ -32,7 +32,15 @@ const StartTrading = () => {
                             )}
                         </Timeline.Item>
                     </Timeline>
-                    <LinkButton mt="4rem" to="/dmt5/" secondary="true">
+                    <LinkButton
+                        mt="4rem"
+                        external="true"
+                        type="mt5"
+                        to="#demo"
```
Comment (sean-binary) [inline src/pages/trade-types/margin/_start-trading.js:39]:
```suggestion
                        to="/#demo"
```

Comment (sean-binary) [reply src/pages/trade-types/margin/_start-trading.js:39]: You will need to include a slash here as the urls from `type` does not end with a trailing slash.

Comment (njazuli-deriv) [reply src/pages/trade-types/margin/_start-trading.js:39]: The format that I need is app.deriv.com/mt5#demo as per card, thus no slash will be added for `type` for now @sean-binary @yashim-deriv

Comment (sean-binary) [reply src/pages/trade-types/margin/_start-trading.js:39]: Oh that's right 👍  my bad.

**Review by sean-binary — APPROVED** [review]

**Pull request merged**